pkg/vere/main.c

```diff
@@ -727,6 +739,7 @@ u3_ve_usage(c3_i argc, c3_c** argv)
     "-p, --ames-port PORT          Set the ames port to bind to\n",
     "    --http-port PORT          Set the http port to bind to\n",
     "    --https-port PORT         Set the https port to bind to\n",
+    "    --snap-time TIME          Set the snapshotting rate in minutes (> 0)\n",
```

Comment:
```suggestion
    "    --snap-time TIME          Set the minutes between snapshots (> 0)\n",
```

Reply:
personally find the original phrasing a bit more clear

Reply:
I like rate better than minutes between as well